charts/refinery/values.yaml

```
@@ -391,8 +391,11 @@ deployMarker:
   image:
     repository: honeycombio/honeymarker
     pullPolicy: IfNotPresent
-    # Overrides the image tag whose default is the chart appVersion.
+    # Overrides the image tag whose default is latest
     tag: "latest"
   honeycombAPI: ""
   honeycombDataset: ""
+  environment: {}
```

```suggestion
  # For now we require an env var name `REFINERY_HONEYCOMB_DEPLOY_MARKER_API_KEY` to contain the api key
  environment: {}
```

Can you include a commented out example of setting the `REFINERY_HONEYCOMB_DEPLOY_MARKER_API_KEY` value with a secret.  I think we do this other places in this file

---

Are you referring to the commented out code on line 400?

---

I mean lets add something like this:

```environment: []
  # - name: REFINERY_HONEYCOMB_DEPLOY_MARKER_API_KEY
  #   valueFrom:
  #     secretKeyRef:
  #       name: honeycomb
  #       key: api-key
```

---

Yes, that's what I have on line 401